operations/node_management/Adding_a_Liquid-cooled_blade_to_a_System.md

```diff
@@ -57,30 +57,52 @@ This procedure will add a liquid-cooled blades from a HPE Cray EX system.
 
 
 ## Procedure
-1.  Suspend the hms-discovery cron job.
+1.  Suspend the hms-discovery cron job to disable it.
 
     ```bash
     ncn-m001# kubectl -n services patch cronjobs hms-discovery -p '{"spec" : {"suspend" : true }}'
     ```
 
-    1.  Verify that the hms-discovery cron job has stopped (`ACTIVE` = `0` and `SUSPEND` = `True`).
+    Verify that the hms-discovery cron job has stopped (`ACTIVE` = `0` and `SUSPEND` = `True`).
 
-        ```bash
-        ncn-m001# kubectl get cronjobs -n services hms-discovery
-        NAME             SCHEDULE        SUSPEND     ACTIVE   LAST   SCHEDULE  AGE
-        hms-discovery    */3 * * * *     True         0       117s             15d
-        ```
+    ```bash
+    ncn-m001# kubectl get cronjobs -n services hms-discovery
+    NAME             SCHEDULE        SUSPEND     ACTIVE   LAST   SCHEDULE  AGE
+    hms-discovery    */3 * * * *     True         0       117s             15d
+    ```
 
-    2.  Power off the chassis slot. This examples powers off slot 0, chassis 3, in cabinet 9000.
+2.  Determine if the destination chassis slot is populated. This example is checking slot 0 in chassis 3 of cabinet x1005.
+    ```bash
```

Comment:
In general, we are trying to separate input/output in separate code blocks to keep it clear where the command ends and output begins in the rendered version of the markdown. This can be applied throughout this procedure. For example...

    ```bash
    ncn-m001# cray hsm state components describe
    ```

    Example output:

    ```
    ID = "x1005c3s0"
    Type = "ComputeModule"
    State = "Empty"
    Flag = "OK"
    Enabled = true
    NetType = "Sling"
    Arch = "X86"
    Class = "Mountain" 
    ```